src/core/surface/call.h

```diff
@@ -86,7 +85,10 @@ typedef void (*grpc_ioreq_completion_func)(grpc_call *call,
                                            void *user_data);
 
 grpc_call *grpc_call_create(grpc_channel *channel, grpc_completion_queue *cq,
-                            const void *server_transport_data);
+                            const void *server_transport_data,
+                            grpc_mdelem **add_initial_metadata,
```

Comment:
Could this be `initial_metadata_to_add` or just `initial_metadata`? I would prefer to not have an array name that looks like a function.